src/game/f3dex3.c

```diff
@@ -0,0 +1,388 @@
+#include <ultra64.h>
+
+#ifdef F3DEX_GBI_3
+#include "game_init.h"
+#ifdef DEBUG_F3DEX3_PROFILER
+#include "buffers/buffers.h"
+#include "puppyprint.h"
+#include "fasttext.h"
+#endif
+
+#ifdef F3DEX3_LIGHTING_ENGINE
+#include "level_update.h"
+#include "engine/math_util.h"
+#include "config/config_world.h"
+#endif
+
+#include "f3dex3.h"
+
+#ifdef DEBUG_F3DEX3_PROFILER
+u32 gF3DEX3ProfilerPage = 0;
+
+volatile F3DEX3YieldDataFooter gRSPProfilingResults;
```

Comment:
No point in `volatile` here. What must be marked as `volatile` is pointer `volatile F3DEX3YieldDataFooter* footer` in `extract_f3dex3_profiler_data`

Reply:
fine for now